android/engine/src/main/java/org/smartregister/fhircore/engine/ui/register/BaseRegisterActivity.kt

```diff
@@ -120,7 +120,8 @@ abstract class BaseRegisterActivity :
         if (!syncTimeStamp.isNullOrEmpty()) {
           SharedPreferencesHelper.write(LAST_SYNC_TIMESTAMP, syncTimeStamp)
         }
-        registerActivityBinding.btnRegisterNewClient.isEnabled = !syncTimeStamp.isNullOrEmpty()
+        // It does not work even if sync is successful
+        // registerActivityBinding.btnRegisterNewClient.isEnabled = !syncTimeStamp.isNullOrEmpty()
```

Comment:
Please uncomment this line and see the issue #540 on how to use the workaround before a fix is implemented

Reply:
@maimoonak Please uncomment this line and see the issue #540 on how to use the workaround before a fix is implemented